modules/nw-multinetwork-sriov.adoc

```diff
@@ -40,247 +62,386 @@ and device ID 0x1015
 * Mellanox MT27800 Family [ConnectX-5] 100G card with vendor ID 0x15b3
 and device ID 0x1017
 
+== Install SR-IOV Network Operator
```

Comment:
Is there any pre-requisite to meet before installing SR-IOV Operator?
For example, we only support SR-IOV deployment on baremetal environment,  might want to call out this as a pre-requisite in a separate section. Other I can think of is when using SR-IOV in userspace mode (DPDK), user might want to configure hugepages which is not supported by SR-IOV Operator itself, worth a link to MCO doc for how to configure kernel args etc.